freqtrade/tests/rpc/test_rpc_telegram.py

```diff
@@ -107,7 +107,7 @@ def test_status_handle(default_conf, update, ticker, mocker):
     _status(bot=MagicMock(), update=update)
 
     assert msg_mock.call_count == 1
-    assert '[BTC_ETH]' in msg_mock.call_args_list[0][0][0]
+    assert '[BTC/ETH]' in msg_mock.call_args_list[0][0][0]
```

Comment:
The slash seems awkward, I haven't seen it used in currency pairs, it seems to always be BTC-ETH or BTC_ETH. Should we stick with that?